test/e2e/cluster.sh

```diff
@@ -146,6 +146,24 @@ else
   exit 0
 fi
 
+docker run --rm \
+    -v $(pwd):${WORK_DIR} \
+    -w ${WORK_DIR} \
+    -e RESOURCE_GROUP=$RESOURCE_GROUP \
+    -e REGION=$REGION \
+    ${DEV_IMAGE} \
+    ./bin/aks-engine rotate-certs \
+    --subscription-id ${AZURE_SUBSCRIPTION_ID} \
+    --api-model _output/$RESOURCE_GROUP/apimodel.json \
+    --apiserver "$RESOURCE_GROUP.$REGION.cloudapp.azure.com" \
+    --location $REGION \
+    --resource-group $RESOURCE_GROUP \
+    --auth-method client_secret \
+    --client-id ${AZURE_CLIENT_ID} \
+    --ssh _output/$RESOURCE_GROUP/$RESOURCE_GROUP-ssh \
```

Comment:
too many `$RESOURCE_GROUP` instances here, should be:

`--ssh _output/$RESOURCE_GROUP-ssh \`